site/content/contribute/server/feature-flags.md

```diff
@@ -99,3 +102,12 @@ There are no hard rules on when a feature flag should be used. It is left up to
 
 8. Does it make sense to use feature flags for A/B testing?
  - This is something we're going to be evaluating using split.io. We have already implemented support for this in the server.
+
+9. Do feature flags need to be restarted?
+ - Feature flags don’t have to be restarted unless the feature requires that.
+
+10. For features that are requested by Self-Managed customers, why do we have to deploy to Cloud first, rather than having the customer who has the test case test it? 
```

Comment:
```suggestion
10. For features that are requested by self-managed customers, why do we have to deploy to Cloud first, rather than having the customer who has the test case test it?
```